scripts/cddl_gen.py

```diff
@@ -896,14 +896,11 @@ def repeated_multi_var_condition(self):
 
     # Whether any extra variables are to be included for this element outside
     # of repetitions.
+    # Also, whether this element must involve a call to multi_xcode(), i.e. unless
+    # it's repeated exactly once.
```

Comment:
Is this more of a side comment, or should it also be a docstring? Same with line 904.

Reply:
It's two different interpretations of the function. I'm planning now to migrate all the function docs from comments to docstrings, but I'll do it in a separate PR, so this could wait until then.